zul/src/archive/metainfo/xml/zul.xsd

```diff
@@ -1017,6 +1017,17 @@
 		<xs:group ref="anyGroup" minOccurs="0" maxOccurs="unbounded" />
 		<xs:attributeGroup ref="xulElementAttrGroup" />
 	</xs:complexType>
+	<!-- barScanner -->
+	<xs:element name="barScanner" type="barScannerType" />
+	<xs:complexType name="barScannerType">
+		<xs:group ref="baseGroup" minOccurs="0" maxOccurs="unbounded" />
+		<xs:attributeGroup ref="xulElementAttrGroup" />
+		<xs:attribute name="type" type="xs:string" use="optional" />
+		<xs:attribute name="contScan" type="booleanType" use="optional" />
+		<xs:attribute name="scanRate" type="doubleType" use="optional" />

```

Comment:
Parameter names don't have to append `scan` again, `contScan` = `continuous`, `scanRate` = `rate`...